File: go/map.go
package main

import "fmt"

func main() {
	got := make(map[int]string)
	got[1] = "a"
	got[2] = "b"
	got[3] = "c"
	fmt.Println()

	fmt.Println("original Map: ", got)
	fmt.Println("KEY\tELEMENT")
	for key, values := range got {
		fmt.Println(key, "\t", values)

	}

	fmt.Println()
	fmt.Println("Let's Updated the ned")
	fmt.Println("KEY\tELEMENT")

	fmt.Println()
	delete(got, 3)
	for key, values := range got {
		fmt.Println(key, "\t", values)

	}
	fmt.Println("Updated Map: ", got)
}

File: c++/map.cpp
#include <iostream>
#include <iterator>
#include <map>
using namespace std;
int main()
{
    map<int, char> map1;
    map<int, char>::iterator cursor;
    map1[1] = 'a';
    map1[2] = 'b';
    map1[3] = 'c';
    cout << "Original MAP" << endl;
    cout << "KEY\tELEMENT" << endl;
    for (cursor = map1.begin(); cursor != map1.end(); cursor++)
    {
        cout << cursor->first;
        cout << '\t' << cursor->second << '\n'
             << endl;
    }
    map1.erase({3});
    cout << "Updated MAP" << endl;
    cout << "KEY\tELEMENT" << endl;
    for (cursor = map1.begin(); cursor != map1.end(); cursor++)
    {
        cout << cursor->first;
        cout << '\t' << cursor->second << '\n'
             << endl;
    }
}